Rakefile

```diff
@@ -29,6 +29,16 @@ def shut_down
   sleep(5)
 end
 
+def connect_db
+  config = YAML.load_file('config/database.yml')
+  config = config[ENV['RACK_ENV']]
+  user, password, host = config['user'], config['password'], config['host']
+  database = config['database']
+
+  puts '[*] Connecting to DB'
+  conn = Mysql2::Client.new(:host => host, :username => user, :password => password, :database => database)
```

Comment:
Great! Please just change your Ruby 1.8 syntax to Ruby 1.9+ 
`Mysql2::Client.new(host: host, username: :user, password: :password, database: :database)`

Reply:
Greate work! I agree with @HugoPouliquen but `conn =` is not needed because Ruby returns the result of the last statement of a method, so the variable is not used.

Reply:
He's referring to the syntax of :host host vs :host => host, etc.

Reply:
Hmm @HugoPouliquen has updated his comment, so mine is deprecated now ;)
And yes I've told to @HugoPouliquen about the Ruby syntax on a private Slack.

Reply:
Thanks, @HugoPouliquen , I've adjusted the syntax.
As I said... not a ruby dev, lol.
The whole "no returns" thing threw me for a loop as well 😐